### ISSUE TITLE
Currently Pull request CI test is failing

### ISSUE THREAD

**Issue by takahirom** [DroidKaigi/conference-app-2020#714]
## Kind (Required)

- Improvement

 ## Overview (Required)

 - We should fix it. And when the test failed. we should use fail status 😭 

- https://github.com/DroidKaigi/conference-app-2020/blob/master/.circleci/config.yml

```
> Task :feature:system:compileDebugUnitTestJavaWithJavac NO-SOURCE
> Task :feature:system:testDebugUnitTest NO-SOURCE

FAILURE: Build completed with 5 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':android-base:lintDebug'.
> Could not resolve all files for configuration ':android-base:androidTestUtil'.
   > Could not resolve androidx.test:orchestrator:1.3.0-alpha02.
     Required by:
         project :android-base
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
      > No cached version of androidx.test:orchestrator:1.3.0-alpha02 available for offline mode.
```

 ## Links	

-

**Comment by takahirom** [comment]
For now, I think it is better to disable --offline.

**Comment by nashcft** [comment]
I'd like to fix this issue, and #706 which is also ignored in pull request CI 🙋‍♂

**Comment by FujiKinaga** [comment]
@nashcft 
Thanks 👍 
For now, I'll assign both issues(this PR and #706 ).
Please let me know if you have any problems🙏
